libraries/rush-lib/src/utilities/Utilities.ts

```diff
@@ -854,39 +866,52 @@ export class Utilities {
 
     const escapedArgs: string[] = args.map((x) => Utilities.escapeShellParameter(x));
 
-    let result: child_process.SpawnSyncReturns<string | Buffer> = child_process.spawnSync(
-      escapedCommand,
-      escapedArgs,
-      options
-    );
+    const process: child_process.ChildProcess = child_process.spawn(escapedCommand, escapedArgs, options);
 
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    if (result.error && (result.error as any).errno === 'ENOENT') {
-      // This is a workaround for GitHub issue #25330
-      // https://github.com/nodejs/node-v0.x-archive/issues/25330
-      //
-      // TODO: The fully worked out solution for this problem is now provided by the "Executable" API
-      // from @rushstack/node-core-library
-      result = child_process.spawnSync(command + '.cmd', args, options);
-    }
+    const stdoutBuffers: string[] = [];
+    process.stdout?.on('data', (chunk) => {
+      stdoutBuffers.push(chunk.toString());
+    });
 
-    Utilities._processResult(result);
-    return result;
-  }
+    const stderrBuffers: string[] = [];
+    process.stderr?.on('data', (chunk) => {
+      stderrBuffers.push(chunk.toString());
+    });
 
-  private static _processResult(result: child_process.SpawnSyncReturns<string | Buffer>): void {
-    if (result.error) {
-      result.error.message += '\n' + (result.stderr ? result.stderr.toString() + '\n' : '');
-      throw result.error;
+    let error: Error | undefined;
+    try {
+      await once(process, 'close');
+    } catch (e) {
+      error = e;
     }
 
-    if (result.status) {
-      throw new Error(
-        'The command failed with exit code ' +
-          result.status +
-          '\n' +
-          (result.stderr ? result.stderr.toString() : '')
-      );
+    const stderr: string = stderrBuffers.join('');
+    const stdout: string = stdoutBuffers.join('');
```

Comment:
I feel like a lot of this should exist under the Executable class... but it may not? (On mobile)